docs/deployment-metrics.md

```diff
@@ -3,6 +3,7 @@
 | Metric name| Metric type | Labels/tags | Status |
 | ---------- | ----------- | ----------- | ----------- |
 | kube_deployment_status_replicas | Gauge | `deployment`=&lt;deployment-name&gt; <br> `namespace`=&lt;deployment-namespace&gt; | STABLE |
+| kube_deployment_status_replicas_ready | Gauge | `deployment`=&lt;deployment-name&gt; <br> `namespace`=&lt;deployment-namespace&gt; | STABLE |
```

Comment:
Could we change this from STABLE to EXPERIMENTAL, promoting is easier in case we change our minds :) Thanks!

Reply:
Done 👍🏽